### ISSUE TITLE
Node.js Technical Steering Committee (TSC) Meeting 2020-08-19

### ISSUE THREAD

**Issue by mhdawson** [nodejs/TSC#910]
## Time

**UTC Wed 19-Aug-2020 14:00 (02:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Wed 19-Aug-2020 07:00 (07:00 AM) |
| US / Mountain | Wed 19-Aug-2020 08:00 (08:00 AM) |
| US / Central  | Wed 19-Aug-2020 09:00 (09:00 AM) |
| US / Eastern  | Wed 19-Aug-2020 10:00 (10:00 AM) |
| London        | Wed 19-Aug-2020 15:00 (03:00 PM) |
| Amsterdam     | Wed 19-Aug-2020 16:00 (04:00 PM) |
| Moscow        | Wed 19-Aug-2020 17:00 (05:00 PM) |
| Chennai       | Wed 19-Aug-2020 19:30 (07:30 PM) |
| Hangzhou      | Wed 19-Aug-2020 22:00 (10:00 PM) |
| Tokyo         | Wed 19-Aug-2020 23:00 (11:00 PM) |
| Sydney        | Thu 20-Aug-2020 00:00 (12:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Technical%20Steering%20Committee%20(TSC)+Meeting+2020-08-19&iso=20200819T14
* or http://www.wolframalpha.com/input/?i=02PM+UTC%2C+Aug+19%2C+2020+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1HcYNwU4Oj0NF2i4j9Youj7umENwZj-AkcZu37uG9n2E/edit>

## Agenda

Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/node

* Land #34035 on v14 (stream: simpler Readable async iterator) [#34680](https://github.com/nodejs/node/issues/34680)
* Rename default branch from "master" to "main" or something similar [#33864](https://github.com/nodejs/node/issues/33864)

### nodejs/admin

* Audit Google account access [#389](https://github.com/nodejs/admin/issues/389)

## Invited

* Anna Henningsen @addaleax (TSC)
* Anatoli Papirovski @apapirovski (TSC)
* Beth Griggs @BethGriggs (TSC)
* Ruben Bridgewater @BridgeAR (TSC)
* Сковорода Никита Андреевич @ChALkeR (TSC)
* Colin Ihrig @cjihrig (TSC)
* Shelley Vohr @codebytere (TSC)
* Daniel Bevenius @danbev (TSC)
* Franziska Hinkelmann @fhinkel (TSC)
* Gabriel Schulhof @gabrielschulhof (TSC)
* Gireesh Punathil @gireeshpunathil (TSC)
* James Snell @jasnell (TSC)
* Joyee Cheung @joyeecheung (TSC)
* Mary Marchini @mmarchini (TSC)
* Matteo Collina @mcollina (TSC)
* Michael Dawson @mhdawson (TSC)
* Myles Borins @MylesBorins (TSC)
* Michaël Zasso @targos (TSC)
* Tobias Nießen @tniessen (TSC)
* Rich Trott @Trott (TSC)

## Notes

The agenda comes from issues labelled with `tsc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

Zoom link: https://zoom.us/j/611357642
Regular password

## Public participation

We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at **https://www.youtube.com/c/nodejs+foundation/live** when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

Many of us will be on IRC in #node-dev on Freenode if you'd like to interact, we have a Q/A session scheduled at the end of the meeting if you'd like us to discuss anything in particular. @nodejs/collaborators in particular if there's anything you need from the TSC that's not worth putting on as a separate agenda item, this is a good place for that.

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

**Comment by mhdawson** [comment]
Since there is nothing new on the list since last week, and I don't think we need additional discussion on the issues which are tagged this week I suggest we cancel.

If I don't hear objections by mid Tuesday my time I'll go ahead and cancel.

**Comment by mmarchini** [comment]
Unhandled rejections survey update: we're at 1400 responses. With the initial target being ~2000 and the last day being Aug 24 (next Monday), we should be able to hit the target without extensions. Will give another update on the issue around Thursday-Friday.

**Comment by ronag** [comment]
> Land #34035 on v14 (stream: simpler Readable async iterator) #34680

FYI. I think this was resolved in the last meeting? At least in terms of what the next step is. I haven't opened a new PR yet.

**Comment by mhdawson** [comment]
@ronag agree, I'll remove the agenda tag.

**Comment by mhdawson** [comment]
@mmarchini awesome to hear that we've already got that many responses.

**Comment by mhdawson** [comment]
No objections to cancelling or new items for the agenda, cancelling.

**Comment by MylesBorins** [comment]
@mhdawson I thought that @jasnell wanted to discuss https://github.com/nodejs/TSC/issues/904

Probably makes sense to discuss next week as this meeting is now cancelled

**Comment by jasnell** [comment]
Next week is fine
